recipes/qtils/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+import os
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import copy,get
+
+required_conan_version = ">=2.0.5"
+
+class PackageConan(ConanFile):
+    name = "qtils"
+    version = "0.0.4"
+    description = "Utils for KAGOME C++ projects"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/qdrvm/qtils"
+    topics = ("KAGOME")
+    package_type = "header-library"
+    settings = "os", "compiler", "build_type", "arch"
+
+    def layout(self):
+        cmake_layout(self)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.generate()
+        deps = CMakeDeps(self)
+        deps.generate()
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def build(self):
+        # Only header files, no source files that need to be compiled, so skip the build step
+        pass
```

Comment:
Missing a `package_id()` that calls `self.info.clear()` as is usually done for header-only libraries